eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
 <fg=cyan>Welcome to eZ Platform!</fg=cyan>
 
 <options=bold>Quick Install:</>
-(Assuming the CLI user you execute commands with below is same that extracted/installed the software)
-<comment>    $  export SYMFONY_ENV="prod"</comment>
+<comment>    $  export SYMFONY_ENV="dev"</comment>
 <comment>    $  php ${consoleDir}/console ezplatform:install <type></comment>
 <comment>    $  php ${consoleDir}/console assetic:dump</comment>
 <comment>    $  php ${consoleDir}/console server:run</comment>
```

Comment:
You can only execute this command if you have `symfony/web-server-bundle` in `composer.json`. And this is not the case in this repo or eZ Platform meta repo.

Reply:
@SylvainGuittard it is there in 2.0: https://github.com/ezsystems/ezplatform/blob/2.0/app/AppKernel.php#L56

Reply:
But not in `composer.json`: https://github.com/ezsystems/ezplatform/blob/2.0/composer.json#L54

Reply:
@SylvainGuittard actually... it is :grinning: 
```bash
$ composer why symfony/web-server-bundle
symfony/symfony  v3.4.0-RC2  replaces  symfony/web-server-bundle (self.version)
```

Reply:
That's very good. I did not know about that. Thanks @alongosz